bin/plugin/lib/utils.js

```diff
@@ -0,0 +1,91 @@
+const inquirer = require( 'inquirer' );
```

Comment:
What's the difference between `plugin/commands/common.js` and `plugins/lib/utils.js`? I also personally find "lib" and "utils" to be rather synonymous, so it's a bit of a confusing hierarchy, especially when some utils are classified ("git" utils).

Reply:
The difference is that the "common" things depend on "config" so they're not completely utilities while "utils" are "pure" (arguments => result).

Maybe it's not the best split, and the naming is definitely not great.